swap_meet/clothing.py

```diff
@@ -1,2 +1,13 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    """
+    A class to represent a Clothing item.
+
+    """
+
+    def __init__(self, condition=0.0):
+        super().__init__(category = "Clothing", condition=condition)
+
+    def __str__(self):
+        return "The finest clothing you could wear."
```

Comment:
👍  Good work in ensuring that all `Clothing` instances have a `category` of `"Clothing"` by passing the value into the parent constructor.